### PR TITLE
feat: support element-plus v1.1

### PR DESCRIPTION
**Pull request by nabaonan**
support min version 1.0.2 and above,  add option called lagency to support 1.0.2

### CODE REVIEW

File: src/resolvers/element-plus.ts [hunk 1]
```diff
@@ -1,31 +1,101 @@
...
 
-    if (name.startsWith('El')) {
-      const partialName = name[2].toLowerCase() + name.substring(3).replace(/[A-Z]/g, l => `-${l.toLowerCase()}`)
-      return {
-        path: `element-plus/es/el-${partialName}`,
-        sideEffects: importStyle ? `element-plus/packages/theme-chalk/src/${partialName}.scss` : undefined,
+    if (name.match(/^El[A-Z]/)) {
+      let sideEffects
+      const partialName = kebabCase(name.slice(2))// ElTableColumn->table-column
+      if (version >= '1.1.0') {
```
[the remark this answers is not in the document]
Comment (nabaonan) [reply src/resolvers/element-plus.ts:79]: Actually, I don't quite understand what's wrong here, could you explain it more clearly?

[the remark this answers is not in the document]
Comment (hannoeru) [reply src/resolvers/element-plus.ts:79]: It's string not a number, you will need something like https://www.npmjs.com/package/compare-versions, or make one by yourself.

[the remark this answers is not in the document]
Comment (nabaonan) [reply src/resolvers/element-plus.ts]: > It's string not a number, you will need something like https://www.npmjs.com/package/compare-versions, or make one by yourself.

Thanks, i have replaced  it .

**Review by antfu — APPROVED** [review]

**Comment by develar** [comment]
Info for everyone interested about what was changed in Element Plus and why: https://github.com/element-plus/element-plus/discussions/3020

**Pull request merged**